account/src/routes.js

```diff
@@ -15,4 +16,20 @@ router.post('/accounts', async (request, response) => {
     }); 
 });
 
+router.post('/tokens', async (request, response) => {
+   const { email, password } = request.body;
+   const authToken = await createUserTokenUseCase(email, password);
+   
+   if(authToken) {
+        return response.status(201).json({
+            token: authToken
+        });
+   }
```

Comment:
i suggest you always put the happy flow at the end

Reply:
adjusted.